miner/worker.go

```diff
@@ -1080,6 +1081,30 @@ func (w *worker) fillTransactions(interruptCh chan int32, env *environment, stop
 			localBlobTxs[account] = txs
 		}
 	}
+
+	if bidTxs != nil {
+		filterBidTxs := func(commonTxs map[common.Address][]*txpool.LazyTransaction) {
+			for acc, txs := range commonTxs {
+				for i := len(txs) - 1; i >= 0; i-- {
+					if bidTxs.Contains(txs[i].Hash) {
+						txs = txs[i+1:]
+						break
+					}
+				}
+				if len(txs) == 0 {
+					delete(commonTxs, acc)
+				} else {
+					commonTxs[acc] = txs
```

Comment:
if no conflict at all, why do we bother to do assign again?

Reply:
cause txs is local value, it's not point to map value

Reply:
fixed